frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -803,14 +804,24 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
     }
 
     private void prepareStandardView(RenderRequest request) throws IOException {
+        User user = UserCacheHolder.getUserFromRequest(request);
         List<Organization> organizations = UserUtils.getOrganizations(request);
         request.setAttribute(PortalConstants.ORGANIZATIONS, organizations);
+        request.setAttribute(IS_USER_ADMIN,isUserAdmin(user) ? "Yes" : "No");
```

Comment:
```suggestion
        request.setAttribute(IS_USER_ADMIN,return PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user) ? "Yes" : "No");
```
and remove the `isUserAdmin` function.

Reply:
Modified the code accordingly.